l10n/en/brands.ftl

```diff
@@ -166,3 +166,7 @@
 -brand-name-google-play = Google Play
 -brand-name-youtube = YouTube
 -brand-name-chromebook = Chromebook
+
+## Mozilla VPN Vendors
+-brand-name-mozilla-vpn-mullvad = Mullvad
```

Comment:
Suggest just calling these `-brand-name-mullvad` and `-brand-name-wireguard`

Reply:
Done.